appium-dotnet-driver/Appium/AppiumCommandExecutionHelper.cs

```diff
@@ -58,6 +65,77 @@ public static void HideKeyboard(IExecuteMethod executeMethod, string strategy =
         public static void Lock(IExecuteMethod executeMethod, int seconds) =>
             executeMethod.Execute(AppiumDriverCommand.LockDevice,
                 new Dictionary<string, object>()
-                    {["seconds"] = seconds});
+                { ["seconds"] = seconds });
+
+        public static string GetClipboard(IExecuteMethod executeMethod, ClipboardContentType clipboardContentType)
+        {
+            return (string)executeMethod.Execute(AppiumDriverCommand.GetClipboard,
+                PrepareArgument("contentType", clipboardContentType.ToString().ToLower())).Value;
+        }
+
+        public static string GetClipboardText(IExecuteMethod executeMethod)
+        {
+            var encodedContentBytes = Convert.FromBase64String(GetClipboard(executeMethod, ClipboardContentType.PlainText));
+            return Encoding.UTF8.GetString(encodedContentBytes);
+        }
+
+        public static string SetClipboard(IExecuteMethod executeMethod, ClipboardContentType clipboardContentType, string base64Content)
+        {
+            switch (clipboardContentType)
+            {
+                case ClipboardContentType.Url:
+                case ClipboardContentType.PlainText:
+                    var encodedContentBytes = Encoding.UTF8.GetBytes(base64Content);
+                    base64Content = Convert.ToBase64String(encodedContentBytes);
+                    break;
+            }
+            return (string)executeMethod.Execute(AppiumDriverCommand.SetClipboard,
+                PrepareArguments(new[] { "content", "contentType", "label" },
```

Comment:
`label` can simply be skipped. There is no need to set it to null

Reply:
Are you saying the label field is optional? if so, If it is null, it shouldn't be sent to the server